scenarios/containermatrix.benchmarks.yml

```diff
@@ -0,0 +1,86 @@
+﻿imports:
+  - https://raw.githubusercontent.com/dotnet/crank/main/src/Microsoft.Crank.Jobs.Bombardier/bombardier.yml
+  - https://github.com/aspnet/Benchmarks/blob/main/scenarios/aspnet.profiles.yml?raw=true
+jobs:
+  aspnetbenchmarks-docker:
```

Comment:
Maybe add a `noClean: true` so it doesn't have to rebuild the docker image.
It's cleaned every night automatically.